packages/core/telemetry/admin/index.js

```diff
@@ -0,0 +1,24 @@
+import { ampli } from './src/ampli/index.js'; // eslint-disable-line
+
+// Loads ampli sdk for the correct environment, checks if processing should be disabled or not
+if (!ampli.amplitude) ampli.load({ environment: 'production' });
+
+const ampliAdminTelemetryMiddleware = async payload => {
+  try {
+    fetch('http://localhost:3300/track', {
```

Comment:
Need to update to the analytics url

Reply:
Admin telemetry is not used at the moment. I missed this url because it's not in use at the moment. I'll revert it